### PR TITLE
Reforged readme

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
📖

### CODE REVIEW

**Comment by stefan-wullems** [comment]
An explicit link to the original repo wouldn’t be too out of place IMO.

**Comment by slinkardbrandon** [comment]
We could possibly make a PR to the original repo to add a link to our org to their README?

File: README.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
+
+> Conquer the command-line. :crossed_swords:
+-------
+> :building_construction: ***This is a Fork aiming at reforging vorpal into typescript, and bringing a 2.0*** :construction:
 
-> Conquer the command-line.
+> :construction_worker: You can join the effort here if you want [:link: on slack](https://join.slack.com/t/vorpal-workspace/shared_invite/enQtNTU3NTU3MTAzNzEyLTE4ODg2YjA1OTg0ZDMyY2I4M2Q3MGZhYjVlZDk1ZDY5NWJiZWFmYjJmYzk1ODdmYmU5ZjAzNGEzNDk5NDFhNWY)
+------
 
 ```text
```
Comment (slinkardbrandon) [inline README.md:19]: We should update this sword to have Reforged in the bottom corner ⚔️

But that doesn’t affect the status of this PR.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-</a>
+[![NPM Version](https://img.shields.io/npm/v/vorpal.svg)](https://www.npmjs.com/package/vorpal)
+[![Status](https://img.shields.io/badge/status-Refactor%20in%20progress-yellow.svg)](https://github.com/vorpaljs-reforged/vorpal/projects/1)
+[![Build Status](https://travis-ci.com/vorpaljs-reforged/vorpal.svg)](https://travis-ci.com/vorpaljs-reforged/vorpal/)
+[![NPM Downloads](https://img.shields.io/npm/dt/vorpal.svg)](https://www.npmjs.com/package/vorpal)
 [![Package Quality](http://npm.packagequality.com/shield/vorpal.svg)](http://packagequality.com/#?package=vorpal)
-<a href="https://www.npmjs.com/package/vorpal">
-  <img src="https://img.shields.io/npm/v/vorpal.svg" alt="NPM Version" />
-</a>
 [![XO code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/sindresorhus/xo)
```
Comment (slinkardbrandon) [inline README.md:9]: Are we sticking with Xo or dropping it in favor of prettier or standardjs?

Could changing our code styles could prevent us from getting merge approval into the original repo? Again, not a blocker on this PR but if we want to swap that out it wouldn’t be a bad time to change it here

File: README.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
+[![Gitter](https://badges.gitter.im/dthree/vorpal.svg)](https://gitter.im/dthree/vorpal)
+
+> Conquer the command-line. :crossed_swords:
+-------
+> :building_construction: ***This is a Fork aiming at reforging vorpal into typescript, and bringing a 2.0*** :construction:
 
-> Conquer the command-line.
+> :construction_worker: You can join the effort here if you want [:link: on slack](https://join.slack.com/t/vorpal-workspace/shared_invite/enQtNTU3NTU3MTAzNzEyLTE4ODg2YjA1OTg0ZDMyY2I4M2Q3MGZhYjVlZDk1ZDY5NWJiZWFmYjJmYzk1ODdmYmU5ZjAzNGEzNDk5NDFhNWY)
+------
```
Comment (slinkardbrandon) [inline README.md:18]: I wouldn’t be opposed to a link back to the original repo but I’m not sure if that’s necessary or not with the breadcrumbs on GitHub. Thoughts?

**Review by slinkardbrandon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AdrieanKhisbe) [reply README.md:19]: @slinkardbrandon I don't get exactly what you mean

File: README.md (hunk 2 above)
Comment (AdrieanKhisbe) [reply README.md:9]: I don't know.
Would like @vorpaljs-reforged/vorpaldevelopers inputs.
But getting ride of the XO badge can be done in a second time

File: README.md (hunk 3 above)
Comment (AdrieanKhisbe) [reply README.md:18]: I can make a badge of it

**Comment by AdrieanKhisbe** [comment]
> An explicit link to the original repo wouldn’t be too out of place IMO.

@Stefanwullems Done in 0b5f4fd

File: README.md (hunk 3 above)
Comment (AdrieanKhisbe) [reply README.md:18]: Done

File: README.md (hunk 2 above)
Comment (Ore4444) [reply README.md:9]: xo can and should be used in addition of prettier. We could replace xo with a different linter, but xo and prettier are mutually exclusive

File: README.md (hunk 1 above)
Comment (Ore4444) [reply README.md:19]: @slinkardbrandon Like this?

**Review by Ore4444 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AdrieanKhisbe) [reply README.md:19]: aaaaaaaah. I see ! (cf [b530d10](/vorpaljs-reforged/vorpal/pull/5/commits/b530d104e440c98b150b97868378821ab1668b79))

File: README.md (hunk 2 above)
Comment (hongaar) [reply README.md:9]: I have no experience with xo yet, so no strong opinion on this. I'd say leave it in for now.

**Review by hongaar — APPROVED** [review]

**Comment by AdrieanKhisbe** [comment]
I think we can go, I'm going to merge it

File: README.md (hunk 2 above)
Comment (slinkardbrandon) [reply README.md:9]: Yeah I have absolutely no experience with xo so I didn’t know if we had to pick one over the other. Leave it for now!

**Review by slinkardbrandon — APPROVED** [review]

**Pull request merged**